x/superfluid/keeper/stake_test.go

```diff
@@ -895,3 +896,104 @@ func (suite *KeeperTestSuite) TestRefreshIntermediaryDelegationAmounts() {
 		})
 	}
 }
+
+func (suite *KeeperTestSuite) TestSuperfluidDelegationGovernanceVoting() {
+	testCases := []struct {
+		name             string
+		validatorStats   []stakingtypes.BondStatus
+		superDelegations [][]superfluidDelegation
```

Comment:
Should we add a field to this test case for setting up normal staking delegations? We need to test that iterate delegations returns both